.github/workflows/CI.yml

```diff
@@ -49,5 +49,5 @@ jobs:
       uses: actions/upload-artifact@013d2b89baa2f354c5ffec54c68bec4ab39a2534 #v3.1.2
       with:
         name: Applications
-        path: io.openliberty.java.internal_fat_${{ matrix.java-version }}/build/libs/io.openliberty.java.internal_fat_${{ matrix.java-version }}.war
+        path: io.openliberty.java.internal_fat_${{ matrix.java-version }}/build/libs/io.openliberty.java.internal_fat_${{ matrix.java-version }}.jar
```

Comment:
Sorry for the confusion Kyle, I did not properly explain what I am really after.  I need a WAR file created, but I need it renamed to have a JAR file extension instead of a WAR file extension.  When WAR files get uploaded to artifactory, they get renamed with JAR file extensions, but DHE does not do this.  That is causing me a big headache to fix, so from now on I just want to have GHA build WAR files that are ostensibly JAR files.

Reply:
Ahh, well under the covers .war and .jar files are just compressed files (aka .zip). 
So we could just add a step to the automation that can generate both :)